### DISCUSSION TITLE
Ideas: Is is good to make PR for every commit?

### DISCUSSION THREAD

**Discussion by noahstoner** [Finalgoal231/finalGoal#9] [About github] [answered]
Discussion is started.

**Comment by noahstoner** [comment]
Of course, that's good.

**Reply by creative2113** [reply]
how?
I don't thinks so.

**Reply by popstar7** [reply]
It make the code clean and smart with review.

**Comment by popstar7** [comment]
It make code better.

**Comment by creative2113** [comment]
I think that we have to think more about that.

**Comment by basitali717** [comment, marked as answer]
Do you want wonderful github account?
If you are, please try to make every PR for every commit.
Then you can raise up your PR percentage and with this others can review your PR.
In the end of that processing, you and others can create issue.
So you can also raise up your issue percentage and can get feedback from other developers.
Let's try to make PR per commit.

**Comment by felipedev418** [comment]
Good. but after PR every time, is there negative effect to projects?
